LenovoLegionToolkit.Lib/Controllers/DisplayBrightnessController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LenovoLegionToolkit.Lib.Controllers
+{
+    public class DisplayBrightnessController
+    {
+        public Task<int> Get()
+        {
+            using var mclass = new ManagementClass("WmiMonitorBrightness")
```

Comment:
Please use the `WMI` class here. No need to duplicate the code.

Reply:
Will do!

Reply:
Using the `WMI` class now. Had to overload a method `CallAsync` because the parameters were not a key pair for the `WmiSetBrightness` method.